docs/guides/high-availability/concepts-memory-and-cpu-sizing.adoc

```diff
@@ -44,8 +44,8 @@ Most CPU time goes into creating new TLS connections, as each client runs only a
 
 * For each 120 refresh token requests per second, 1 vCPU to the cluster (tested with up to 435 refresh token requests per second).
 
-* Leave 200% extra head-room for CPU usage to handle spikes in the load.
-This ensures a fast startup of the node, and sufficient capacity to handle failover tasks like, for example, re-balancing Infinispan caches, when one node fails.
+* Leave 125% extra head-room for CPU usage to handle spikes in the load.
```

Comment:
I am wondering what was the reason for the 200% head-room with A/P setup. If it was necessary there, maybe it is still needed for A/A as well. We should probably wait for @ahus1 's opinion

Reply:
I don't remember where we got the 200%.
With A/P, you should not need any head-room at all. 
With A/A it would make sense to double the CPU count in case one of the sites crashes, the other needs to be able to handle the load. In any case, 200% seems way too pessimistic.

Reply:
I agree with you and 125% makes sense to me. Maybe the reason for the extra headroom is some unexpected load, like some peak period but I am not sure that is something we should document, it is probably something the users should consider themselves based on their use-case. Let's wait for @ahus1. From my perspective this change is ok.